packages/frontend/@n8n/design-system/src/components/N8nMenuItem/MenuItem.vue

```diff
@@ -261,12 +310,53 @@ const N8nMenuItem = getCurrentInstance()?.type;
 		padding: var(--spacing-2xs) 0 !important;
 		justify-content: center;
 	}
+
+	&.small {
+		font-size: var(--font-size-2xs) !important;
```

Comment:
Yes, these `!important`s on this file hurt me too :(

Reply:
Yeah but looks like it's needed with all of the other `!important`s already there :/ We need to clean this up